app/src/routes/v3/forest-watcher.router.js

```diff
@@ -274,6 +274,11 @@ class ForestWatcherRouter {
     await AreaTemplateRelationService.delete(ctx.request.params);
     ctx.status = 200;
   }
+
+  static async getAreaTeams(ctx) {
+    ctx.body = await AreaTeamRelationService.getAllTeamsForArea(ctx.request.params);
+    ctx.status = 200;
```

Comment:
I can't see any error handling in [AreaTeamRelationService](https://github.com/wri/fw_api/blob/de3c533d281bbef3835e126b152dd938f3a4d7a2/app/src/services/areaTeamRelationService.js#L13).

And as you're setting the status to 200 after you call the service the API will always return an okay response. Need an 404 response I think.